doc/tutorial/single_cell_model.rst

```diff
@@ -53,40 +53,43 @@ create the ``cable_cell`` that represents it are as follows:
+The segment tree is the representation used to construct the morphology of a cell. A segment is
+a tapered cone with a tag; the tag can be used to classify the type of the segment (for example
+soma, dendrite etc). To create a segment tree representing our single-cylinder cell, we need to add
+one segment to our ``tree`` object. We use the :meth:`arbor.segment_tree.append` method, which takes
+4 arguments: the parent segment which does not exist for the first segment, so we use :class:`arbor.mnpos`;
+the proximal :class:`arbor.mpoint` (location and radius) of the segment; the distal :class:`arbor.mpoint`
+of the segment; and the tag.
 
-In step **(2)** a dictionary of labels is created using :class:`arbor.label_dict<arbor.label_dict>`.
-Cell builders need to refer to *regions* and *locations* on a cell morphology. Arbor uses a domain
-specific language (DSL) to describe regions and locations, which are given labels. We add two labels:
+Step **(2)** creates a dictionary of labels (:class:`arbor.label_dict<arbor.label_dict>`). Labels give
+names to :ref:`regions<labels-region>` and :ref:`location<labels-locset>` described using a DSL
+based on s-expressions. Labels from the dictionary can then be used to facilitate adding synapses,
+dynamics, stimulii and probes to the cell. We add two labels:
 
 * ``soma`` defines a *region* with ``(tag  1)``. Note that this corresponds to the
   ``tag`` parameter that was used to define the single segment in step (1).
 * ``center`` defines a *location* at ``(location 0 0.5)``, which is the mid point ``0.5``
   of branch ``0``, which corresponds to the center of the soma on the morphology defined in step (1).
 
-In step **(3)** a :class:`arbor.cable_cell` is constructed by combining the segment tree
-with the named regions and locations.
-
-* "Cell-wide" properties are set through :meth:`arbor.cable_cell.set_properties`. Here,
-  the initial membrane potential is set to -40 mV everywhere on the cell.
-* Properties can also be set to a region of the cell, which Arbor calls 'painting'. This
-  is meant to convey placement is not precise: we wouldn't want to manually place ion
-  channels all over the surface of the cell. :meth:`arbor.cable_cell.paint` lets us
-  instruct Arbor to use HH dynamics on the region we've labelled soma and sort the details
-  out for us.
-* Other properties should be added to the cell on a precise :class:`arbor.location`. This is
-  done using the :meth:`arbor.cable_cell.place<arbor.cable_cell.place>` method.
-  We place a current stimulus :class:`arbor.iclamp<arbor.iclamp>` with a duration of 2 ms
-  and a current of 0.8 nA, starting at 10 ms on the location we've labelled 'center'. We also
-  place a :class:`arbor.spike_detector<arbor.spike_detector>` with a threshold of -10 mV on the
-  same location.
+Step **(3)** constructs the :class:`arbor.cable_cell` from the segment tree and dictionary of labeled
```

Comment:
This will change with the new cable_cell, right? So this part of the example will need an update.

Reply:
Yes. Again, I'm just trying to keep the information correct for the current state of Arbor. We can change it again when the new changes are in.

Reply:
OK.